ddtrace/bootstrap/sitecustomize.py

```diff
@@ -85,7 +85,7 @@ def drop(module_name):
             _unloaded_modules.append(sys.modules[module_name])
         del sys.modules[module_name]
 
-    MODULES_REQUIRING_CLEANUP = ("gevent",)
+    MODULES_REQUIRING_CLEANUP = ("gevent", "multiprocessing")
```

Comment:
This is essentially equivalent to setting `DD_UNLOAD_MODULES_FROM_SITECUSTOMIZE` to `true` by default as `multiprocessing` is part of the stdlib

Reply:
Good point. I'll also set ``DD_UNLOAD_MODULES_FROM_SITECUSTOMIZE`` to ``true`` by default. I think that would make this change easier to follow